backend/.env.template

```diff
@@ -59,3 +59,5 @@ EVENT_PROTOCOL_DISABLED=$EVENT_PROTOCOL_DISABLED
 # Federation
 FEDERATION_DHT_TOPIC=$FEDERATION_DHT_TOPIC
 FEDERATION_DHT_SEED=$FEDERATION_DHT_SEED
+FEDERATION_DHT_TEST_SOCKET=$FEDERATION_DHT_TEST_SOCKET
```

Comment:
Why do this on the same branch when its clearly out of scope from the original ticket?

Reply:
without a test infrastructure of the DHT hyperswarm and socket communication, on my opinion a valid interims solution to test different data constellations by configuring one instance of the swarm in DHT_TEST_SOCKET-Mode